test/models/invitation_test.rb

```diff
@@ -22,4 +23,26 @@ def teardown
       @event.destroy
     end
   end
+
+  test 'requires email' do
+    invitation = Invitation.new
+    assert_not invitation.valid?
+    assert_equal ["can't be blank"], invitation.errors[:email]
+  end
+
+  test 'allows invitations sharing email for different events' do
+    invitation = Invitation.create(event: @event, email: 'invited@email.com')
+    alt_event = Event.new(name: 'Different event', owner: @user, date: Time.zone.now)
+    new_invitation = Invitation.new(event: alt_event, email: 'invited@email.com')
+
+    assert new_invitation.valid?
+  end
+
+  test 'does not allow invitation sharing email for same event' do
+    invitation = Invitation.create(event: @event, email: 'invited@email.com')
+    new_invitation = Invitation.new(event: @event, email: 'invited@email.com')
+
+    assert_not new_invitation.valid?
+    assert_equal ["is already taken"], new_invitation.errors[:email]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.